geps/gep-1742.md

```diff
@@ -1,7 +1,7 @@
 # GEP-1742: Timeouts
 
 * Issue: [#1742](https://github.com/kubernetes-sigs/gateway-api/issues/1742)
-* Status: Provisional
+* Status: Implementable
```

Comment:
I've been considering if we should let this one soak for just a _bit_ longer in Provisional after this PR: what are your thoughts and feelings on changing the status in a separate follow-up PR to allow just a little bit more time for things to settle, would that be painful?